test/rebar3_hex_integration_SUITE.erl

```diff
@@ -12,18 +12,8 @@ all() ->
     [ sanity_check
```

Comment:
A good bit of clean up was done in here, mostly moving the `mock_command` call to `setup_state/2` in favor of just providing a `#{command => #{provider => atom(), args => list()}` structure for specifying what you want, not how you want it.